include/zephyr/net/http/server.h

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2023, Emna Rekik
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_NET_HTTP_SERVER_H_
+#define ZEPHYR_INCLUDE_NET_HTTP_SERVER_H_
+
+#include <stdint.h>
+
+#if defined(CONFIG_POSIX_API)
+
+#include <stddef.h>
+#include <poll.h>
+
+#else
+
+#include <zephyr/posix/poll.h>
+
+#endif
+
+#define CLIENT_BUFFER_SIZE         CONFIG_NET_HTTP_SERVER_CLIENT_BUFFER_SIZE
```

Comment:
These should probably not go in this header.

Reply:
Yes, agreed. I think we need to create internal http server header file for these.